autoload/lsp/diag.vim

```diff
@@ -162,14 +193,24 @@ def DiagsRefresh(bnr: number)
       endif
 
       if opt.lspOptions.showDiagWithVirtualText
+
         var padding = diag.range.start.character
         if padding > 0
           padding = strdisplaywidth(getline(diag.range.start.line + 1)[ : diag.range.start.character - 1])
         endif
+
+        var symbol = diag_symbol
+
+        if diag_align ==? 'after'
```

Comment:
Here `diag_align` is set from within the source code, so there cannot be any need for `==?` 😅